### DISCUSSION TITLE
Zeffris in Sonic Pi

### DISCUSSION THREAD

**Discussion by beryann** [amiika/ziffers#83] [Q&A]
Hello! Ziffers notation in Sonic Pi is great but quite difficult to understand and assimilate....There is here of course the Github documentation but is there an another tutorial which explains step by step all the subtilities of it? Thanks for reading! Best Regards!

**Comment by Bubobubobubobubo** [comment]
It is helpful to keep this document not too far from your view when experimenting: https://github.com/amiika/ziffers/blob/master/Cheatsheet.pdf

**Reply by beryann** [reply]
Merci Raphael....sommes en contact sur discord(crash server) et scsynth.....ok pout la Cheatsheet que j'ai d'ailleurs imprimé et que je vais effectivement consulté en même temps que le wiki....j'utilise beaucoup Sonic Pi et cette (mini) notation est vraiment intéressante mais parfois un peu complexe à saisir dans le wiki de Github. Est-il prévu par la suite d'autres tutoriels vidéos ou PDF allant dans le sens d'un manuel utilisateur plus progressif peut-être? Merci de ton attention!

**Reply by Bubobubobubobubo** [reply]
Salut, je réponds en français pour cette fois mais il est préférable de maintenir le dialogue en anglais pour @amiika. Je ne sais pas très bien ce qui est prévu pour le projet, je ne suis pas impliqué directement dans le développement. Je pense qu'il est possible d'approcher la complexité graduellement en travaillant chaque exemple isolément. La cheatsheet fait office de manuel pour l'utilisateur pour le moment. Regarde aussi le dossier des exemples disponibles directement dans le repo.

Back to english now 😄

**Reply by beryann** [reply]
Ok Bubo ;thanks for your quick answer....I'll do so!! Have a nice day! Beryann
